packages/dashmate/README.md

```diff
@@ -226,23 +226,34 @@ The `reset` command removes all data corresponding to the specified config and a
 ```
 USAGE
   $ dashmate reset [--config <value>] [-v] [-h] [-f] [-p]
-
 FLAGS
   -f, --force          skip running services check
   -h, --hard           reset config as well as data
   -p, --platform-only  reset platform data only
   -v, --verbose        use verbose mode for output
   --config=<value>     configuration name to use
-
 DESCRIPTION
   Reset node data
 ```
 
 To reset a node:
 ```bash
 $ dashmate reset
+=======
+#### Hard reset
```

Comment:
I thought this was intended to be its own subsection (like "Manual reset" below) since it used the markdown heading syntax. Is that not the case?

Reply:
It makes sense. Check if now it looks better for you :)

Reply:
Looks good :+1: This is what I was attempting to do originally, but I messed it up somehow :smile: